components/calendar/src/astronomy.rs

```diff
@@ -488,7 +575,12 @@ impl Astronomical {
         Self::universal_from_dynamical(approx + correction + extra + additional)
     }
 
-    #[allow(dead_code)]
+    /// Sidereal time, as the hour angle between the meridian and the vernal equinox,
+    /// from a given moment.
+    ///
+    /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz.
```

Comment:
Whenever the Lisp code has a citation, please preserve that citation; there is valuable context in _Astronomical Algorithms_ (for instance it tells us that the input here is in UT, by which Meeus means what we would now call UT1; should UTC start diverging from UT1, we may need to do something).
```suggestion
    /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz,
    /// in turn from _Astronomical Algorithms_ by Meeus, 2nd edition (1988), p. 88.
```

(Several other functions should likewise have a Meeus citation.)